pipeline/pipeline.go

```diff
@@ -181,10 +181,9 @@ func New(name string, settings *Settings, registry *prometheus.Registry) *Pipeli
 
 	var eventPool pool
 	switch settings.Pool {
-	case PoolTypeStd, "":
+	case PoolTypeStd:
 		eventPool = newEventPool(settings.Capacity, settings.AvgEventSize)
-	case PoolTypeLowMem:
-		insaneJSON.StartNodePoolSize = 16
```

Comment:
We can change this value to find a compromise value between 16 (low_mem default value) and 1024 (std default value).